nuitka/utils/CStrings.py

```diff
@@ -28,34 +28,31 @@ def _encodePythonStringToC(value):
     """
     assert type(value) is bytes, type(value)
 
-    result = ""
+    special_chars = {92: r"\134", 9: r"\011", 13: r"\015", 10: r"\012", 63: r"\077", 34: r"\042"}
+    result = ['"']
     octal = False
-
     for c in value:
-        if str is bytes:
-            cv = ord(c)
-        else:
-            cv = c
-
-        if c in b'\\\t\r\n"?':
-            result += r"\%03o" % cv
+        cv = c
```

Comment:
This removes Python2 compatibility, there `ord(c)` is really needed. For performance, of this individual function, giving up on detecting the need inside a loop would probably be best.